Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/publicwebsite/AboutTextManager.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.spring.quadriga.service.impl.publicwebsite;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.asu.spring.quadriga.dao.publicwebsite.IAboutTextDAO;
+import edu.asu.spring.quadriga.dto.AboutTextDTO;
+import edu.asu.spring.quadriga.service.publicwebsite.IAboutTextManager;
+
+/**
+ * Service to save title and description of public website about page form.
+ * 
+ * @author Rajat Aggarwal
+ *
+ */
+@Service
+public class AboutTextManager implements IAboutTextManager {
+
+	@Autowired
+	private IAboutTextDAO aboutTextDAO;
+
+	@Transactional
+	@Override
+	public void saveAbout(String projectId, String title, String description) {
+		AboutTextDTO aboutTextDTO = aboutTextDAO.getDTOByProjectId(projectId);
+
+		if (aboutTextDTO == null)
+			aboutTextDTO = new AboutTextDTO();
+
+		aboutTextDTO.setProjectId(projectId);
+		aboutTextDTO.setTitle(title);
+		aboutTextDTO.setDescription(description);
+		aboutTextDAO.saveNewDTO(aboutTextDTO);
+	}
+
+	@Transactional
+	@Override
+	public AboutTextDTO getDTOByProjectId(String projectId) {
```

Comment:
Nope. You manager can retrieve DTOs from the DAO, but it should not pass them on. You should return an interface here for a domain class. If you domain class is the same as the dto, you can have your dto implement the interface and still return the dto, but classes using the manager shouldn't know that they get DTOs.